frontend/src/pages/org/workflow-editor.ts

```diff
@@ -213,6 +215,7 @@ const DEFAULT_BEHAVIORS = [
   "autofetch",
   "siteSpecific",
 ];
+const BYTES_PER_GB = 1073741824;
```

Comment:
We should decide if we want to do GB or GiB? I wonder if GiB might be more user-friendly, but either way can do a consistency pass on this later.